### PR TITLE
[QAA] adding Aptos send test

### PR DESCRIPTION
**Pull request by VicAlbr**
### ✅ Checklist

- [ ] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Add e2e test

### ❓ Context

- **JIRA or GitHub link**: [https://ledgerhq.atlassian.net/browse/LIVE-14967](https://ledgerhq.atlassian.net/browse/LIVE-14967)

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Comment by VicAlbr** [comment]
@hedi-edelbloute going to close the PR

File: apps/ledger-live-desktop/tests/specs/speculos/send.tx.spec.ts [hunk 1]
```diff
@@ -191,6 +191,10 @@ const transactionE2E = [
     transaction: new Transaction(Account.XRP_1, Account.XRP_2, "0.0001"),
     xrayTicket: "B2CQA-2816",
   },
+  {
+    transaction: new Transaction(Account.APTOS_1, Account.APTOS_2, "0.0001"),
+    xrayTicket: "",
```
[the remark this answers is not in the document]
Comment (semeano) [reply apps/ledger-live-desktop/tests/specs/speculos/send.tx.spec.ts:196]: I think we need a ticket per test type. If you check the previous one for XRP (https://ledgerhq.atlassian.net/browse/B2CQA-2816) it's related only with the "send" test.

AFAIK we don't have those tickets yet.

[the remark this answers is not in the document]
Comment (ypolishchuk-ledger) [reply apps/ledger-live-desktop/tests/specs/speculos/send.tx.spec.ts:196]: Please ask QA of your squad to provide you with test case ticket to attach here

[the remark this answers is not in the document]
Comment (jccguimaraes) [reply apps/ledger-live-desktop/tests/specs/speculos/send.tx.spec.ts:196]: Hi @ypolishchuk-ledger we don't have an assigned QA, but we will find out thanks

[the remark this answers is not in the document]
Comment (ypolishchuk-ledger) [reply apps/ledger-live-desktop/tests/specs/speculos/send.tx.spec.ts:196]: @mcoursier-ledger can you help on this one please?

[the remark this answers is not in the document]
Comment (mcoursier-ledger) [reply apps/ledger-live-desktop/tests/specs/speculos/send.tx.spec.ts:196]:
```suggestion
    xrayTicket: "B2CQA-2920",
```

**Review by mcoursier-ledger — CHANGES_REQUESTED** [review]

**Review by ypolishchuk-ledger — APPROVED** [review]

**Pull request merged**